composer.json

```diff
@@ -24,7 +24,8 @@
     "aws/aws-sdk-php-symfony": "^2.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^6.0|^7.0"
+    "phpunit/phpunit": "^6.0|^7.0",
+    "doctrine/collections": "^1.5"
```

Comment:
This bundle is used within Symfony that already sets dependencies and versions correctly for doctrine/collections.
This requirement is causing the failures. Please, could you remove it.